bin/glite-info-update-endpoints

```diff
@@ -228,11 +252,11 @@ if __name__ == "__main__":
 
     egi_urls = None
     if config['EGI']:
-        if not config['certification_status'] in ["Candidate", "Uncertified", 
-                                                  "Certified", "Closed", 
+        if not config['certification_status'] in ["Candidate", "Uncertified",
```

Comment:
Not to be pedantic, but I think it would be more readable to have the statuses one per line, or even better in an external configuration file. Suggestion for next sprint.

Reply:
Indeed some things are not covered by pep8 and OpenStack guidelines, some even more uglier were passing the tests, like:
```python
plop = "What is this %s" % ("toglut",)
```